sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -14,13 +16,24 @@
   - ShareClient::BreakLease
   - ShareClient::RenewLease
 
+### Breaking Changes
+
+* CreateFromConnectionString now accepts unencoded file and directory name.
+* Added support for getting range list with previous snapshot. `GetFileRangeListResult` now returns `std::vector<FileRange> Ranges` and `std::vector<FileRange> ClearRanges` instead of `std::vector<Range> RangeList`.
+* Added support for SMB Multi-Channel setting for `ServiceClient::GetProperties` and `ServiceClient::SetProperties`. This is only available for Storage account with Premium File access.
+  - Standard account user has to remove the returned SMB Multi-Channel setting before set, otherwise service would return failure.
```

Comment:
How is this a breaking change?

Reply:
This is a behavioral breaking change. Some code used to work will not be working anymore after this change.
Imagine the scenario where a customer retrieves service properties and set it back without any modification. It used to work but not anymore.